qemu/tests/cfg/smbios.cfg

```diff
@@ -5,6 +5,10 @@
     type = smbios_table
     requires_root = yes
     start_vm = no
+    # Remove extra pci_controllers as they only make sense for q35 but we
+    # might be booting i440fx machines as well which would require PCIE
+    # bridge
+    pcie_extra_root_port = 0
```

Comment:
Yes, will get error on plugging pcie-root-port to pci.0 when boot with `-machine pc-i440fx-rhel7.6.0` in test case `smbios_table.machine_type.q35`.